code/modules/cm_marines/overwatch.dm

```diff
@@ -245,7 +245,9 @@
 					else
 						dat += "<font color='green'>Ready!</font><br>"
 					dat += "<B>Launch Pad Status:</b> "
-					var/obj/structure/closet/crate/C = locate() in current_squad.drop_pad.loc
+					var/obj/C = locate() in current_squad.drop_pad.loc //This thing should ALWAYS exist.
+					if(!istype(C,/obj/machinery/vending) && !istype(C,/obj/structure/closet/crate)) //Can only send vendors and crates
```

Comment:
This is a big hack. Sending down a vending machine?

Reply:
What's the issue?

Reply:
I mean if you want to restrict it to the lasgun recharger, that's fine; this exception is largely here for that, though medvends on supply drop would be pretty nice.